### PR TITLE
Remove first part of the update description

### PR DESCRIPTION
**Pull request by ludeeus**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by spacegaier — DISMISSED** [review]

File: src/translations/en.json [hunk 1]
```diff
@@ -4180,7 +4180,7 @@
       "update_name": "Update {name}",
       "open_release_notes": "Open release notes",
       "create_backup": "Create backup before updating",
-      "description": "There is an update available for the {name}. You have {version} installed. Click update to update to version {newest_version}",
+      "description": "There is an update available for {name}. You have {version} installed. Click update to update to version {newest_version}",
```
Comment (balloob) [inline src/translations/en.json:4183]: I think that we should split this in 2 separately translated pieces. 

- One per type that has an update available: "There is an update available for Home Assistant core." etc
- You have {version} installed. Click update to update to version {newest_version}."

Comment (ludeeus) [reply src/translations/en.json:4183]: In that case, I think we should drop the first part ("There is an update available for the {name}. ") entirely.
We are already in an update context.

Comment (balloob) [reply src/translations/en.json:4183]: People need to see _somewhere_ on the page what they are updating though.

Comment (balloob) [reply src/translations/en.json:4183]: image

You're right. Let's drop it.

**Review by balloob — APPROVED** [review]

**Pull request merged**